cmp/grid/Grid.js

```diff
@@ -201,6 +201,7 @@ class LocalModel {
             },
             popupParent: document.querySelector('body'),
             suppressAggFuncInHeader: true,
+            suppressMakeColumnVisibleAfterUnGroup: true,
```

Comment:
I thought a bit more about this -- let's just apply this now in DataView's agOptions, where we know we absolutely want this, and that will allow us to move forward with our DataView enhancements.

Think changing Grid default itself needs to be done with a bit more care, testing, and consideration of what we want for our grouping API.

Reply:
Great, I moved it to DataView only and its localModel now accepts props and agOptions defined as PT to allow for overriding of the suppressMakeColumnVisibleAfterUngroup prop